src/halmos/__main__.py

```diff
@@ -1124,14 +947,36 @@ def parse_build_out(args: argparse.Namespace) -> Dict:
     return result
 
 
+def parse_devdoc(funsig: str, contract_json: Dict) -> str:
+    try:
+        return contract_json["metadata"]["output"]["devdoc"]["methods"][funsig][
+            "custom:halmos"
+        ]
+    except Exception as err:
```

Comment:
catching Exception here as a shorthand because any of these keys might be missing, right?

Reply:
might want to catch KeyError to avoid overcatching

Reply:
yes